nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pipewire.wireplumber;
+in
+{
+  meta.maintainers = [ maintainers.k900 ];
+
+  options = {
+    services.pipewire.wireplumber = {
+      enable = mkEnableOption "A modular session / policy manager for PipeWire";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.wireplumber;
+        defaultText = literalExpression "pkgs.wireplumber";
+        description = ''
+          The wireplumber derivation to use.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    systemd.packages = [ cfg.package ];
+
+    systemd.services.wireplumber.enable = config.services.pipewire.systemWide;
+    systemd.user.services.wireplumber.enable = !config.services.pipewire.systemWide;
+
+    systemd.services.wireplumber.wantedBy = [ "pipewire.service" ];
+    systemd.user.services.wireplumber.wantedBy = [ "pipewire.service" ];
+
+    assertions = [
```

Comment:
Assertions typically are listed first but should not be a big deal in such a small module.

Reply:
Fixed anyway.